pcweb/pages/docs/getting_started/introduction.py

```diff
@@ -24,12 +70,25 @@ def decrement(self):
         self.count -= 1
 
 
-def intro1() -> rx.Component:
+
+def get_started() -> rx.Component:
     return rx.fragment(
-        docheader(text="Introduction", first=True),
-        text_comp(
-            "Reflex is a full-stack framework for building and deploying web apps."
+        subheader_comp(text="Get Started"),
+        doctext(
+            "Below is a quick example of a counter app to get a feel for how Reflex works. ",
         ),
+        doctext(
+            "For a more in depth example, we recommend going through the ",
+            doclink("tutorial", href="/docs/quickstart"),
```

Comment:
I think this is the wrong link - we should import the page i.e. `from ...tutorial import intro` and reference the path as `intro.path` so that the links always stay correct